.github/workflows/build-os-image.yml

```diff
@@ -575,6 +575,7 @@ jobs:
         shell: bash
         run: |
           curl -fsSLO https://github.com/sigstore/rekor/releases/download/v0.12.0/rekor-cli-linux-amd64
+          echo 23c9baefe12b28879275f5aa84dbe9cd734d67ba42bf9bbb28a3eedbf1ed2dd4  rekor-cli-linux-amd64 | sha256sum -c --strict
```

Comment:
This is obsolete with #2798, where we install cosign and rekor-cli via nix.

Reply:
I'm reverting it because I need to fix conflicts anyway.